meson.build

```diff
@@ -7,17 +7,17 @@ blas_deps = []
 if host_machine.system() == 'darwin'
   blas_deps = [dependency('Accelerate')]
 else
-  blas_deps = [cc.find_library('openblas', static: get_option('link_blas_statically'), required : false)]
+  blas_deps = [dependency('openblas', static: get_option('link_blas_statically'), required : false)]
 endif
 
 # try to find blas/cblas (e.g., Linux)
 if not blas_deps[0].found()
-    blas_deps = [cc.find_library('blas', static: get_option('link_blas_statically'), required : false)]
-    lapack_dep = cc.find_library('lapack', static: get_option('link_blas_statically'), required : false)
+    blas_deps = [dependency('blas', static: get_option('link_blas_statically'), required : false)]
+    lapack_dep = dependency('lapack', static: get_option('link_blas_statically'), required : false)
     if lapack_dep.found()
         blas_deps += lapack_dep
     endif
-    cblas_dep = cc.find_library('cblas', static: get_option('link_blas_statically'), required : false)
+    cblas_dep = dependency('cblas', static: get_option('link_blas_statically'), required : false)
     if cblas_dep.found()
         blas_deps += cblas_dep
     endif
```

Comment:
Should we remove the cc.find_library on line 29 too?

Reply:
@bodono I wouldn't touch the one on 29, it's very fragile it's the one that's made to work with the setup in CI to build windows wheels, it is not meant to be used in other settings

Reply:
OK fair enough.